app/assets/templates/comment-stream_tpl.jst.hbs

```diff
@@ -26,7 +26,14 @@
 
     <div class="bd">
       <form accept-charset="UTF-8" action="/posts/{{id}}/comments" class="new_comment" id="new_comment_on_{{id}}" method="post">
-        <textarea class="comment_box form-control" id="comment_text_on_{{id}}" name="text" rows="1" required placeholder="{{t "stream.comment"}}" />
+        <div class="mentions-input-box">
+          <div class="mentions-box">
+            <div class="mentions"/>
+          </div>
+          <textarea class="comment_box form-control mention-fake-textarea"
```

Comment:
I need some help from frontend people here. Since `textarea` went down in the hierarchy here, `submit_button` is not a sibling anymore, and therefore [this](https://github.com/diaspora/diaspora/blob/dba8de54888b50291c5b7b4e64af1fd6c03e2e19/app/assets/stylesheets/comments.scss#L63) style doesn't apply anymore. Does it mean that the only option I have is to implement that behavior (submit button hiding) via JS (like we do for the publisher)?